File: clojure/002/002.clj
(defn fibs_less_than
  ([x]
    (fibs_less_than x '(1 0)))
  ([x, fibs]
    (if (> (+ (first fibs) (second fibs)) x)
      fibs
      (fibs_less_than x (conj fibs (+ (first fibs) (second fibs))))
    )
  )
)

(println (apply + (filter (fn [x] (= (mod x 2) 0)) (fibs_less_than 4000000))))

File: julia/002/002.jl
fibs = [0, 1]

while fibs[length(fibs)] + fibs[length(fibs) - 1] < 4000000
    append!(fibs, [fibs[length(fibs)] + fibs[length(fibs) - 1]])
end

println(sum(filter(x -> mod(x, 2) == 0, fibs)))
